ui/app/templates/storage/volumes/volume.hbs

```diff
@@ -7,6 +7,7 @@
   <Breadcrumb @crumb={{crumb}} />
 {{/each}}
 {{page-title "CSI Volume " this.model.name}}
+{{!-- TODO: determine if /volumes/volume will just be CSI or if we ought to generalize it --}}
```

Comment:
DHV volumes don't have as much detail as CSI volumes. But they do have some detail that's probably not going to want to be in the list view.